.github/workflows/test.yml

```diff
@@ -321,7 +321,7 @@ jobs:
             - if: ${{ needs.release-candidate-deploy.outputs.should_run_tests && !contains(github.head_ref, 'release--') && !(github.ref == 'refs/heads/main' && contains(github.event.head_commit.message, 'Merge pull request') && contains(github.event.head_commit.message, 'demergent-labs/release--')) }}
               shell: bash -l {0}
               working-directory: ${{ matrix.example_directories }}
-              run: AZLE_PROPTEST_NUM_RUNS=5 AZLE_PROPTEST_VERBOSE=true npm test
+              run: AZLE_PROPTEST_NUM_RUNS=5 AZLE_PROPTEST_VERBOSE=true AZLE_INTEGRATION_TEST_LINK_AZLE=true npm test
```

Comment:
@lastmjs we should discuss this as part of the review

Reply:
"I'm not sure you can just do this generally, we need to handle the npm and repo tests. There are tests that run with Azle installed from npm during release, not from the repo, those must be honored correctly. We should discuss." - @lastmjs

Reply:
We are testing this now

Reply:
We will remove these logs in a subsequent pr, testing in a release ci/cd